cpp/src/slice2cs/Gen.cpp

```diff
@@ -1775,10 +1778,19 @@ Slice::Gen::TypesVisitor::visitStructEnd(const StructPtr& p)
 
     if(partialInitialize)
     {
-        _out << sp << nl << "partial void Initialize();";
+        _out << sp;
+        _out << nl << "/// <summary>The generated constructor calls the Initialize partial method after initializing "
+             << "the struct data members.</summary>";
+        _out << nl << "partial void Initialize();";
     }
 
     _out << sp;
+    _out << nl << "/// <summary>Constructs a new instance of the <see cref=\"" << name << "\"/> struct type.</summary>";
```

Comment:
Is it really necessary to say `... struct type.`?
```suggestion
    _out << nl << "/// <summary>Constructs a new instance of <see cref=\"" << name << "\"/> .</summary>";
```